frontend-opal/src/components/common/AddTabs.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Tab, Tabs } from 'native-base';
+import { StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+
+import tools from './tools';
+
+const styles = StyleSheet.create({
+  tabs: {
+    zIndex: 10
+  },
+  tabStyle: {
+    backgroundColor: 'white'
+  },
+  textStyle: {
+    color: 'rgb(250,110,59)'
+  },
+  activeTabStyle: {
+    backgroundColor: 'white',
+    borderBottomWidth: tools.getPixel(2),
+    borderBottomColor: 'rgb(250,110,59)'
+  }
+});
+
+function AddTabs({ Cmpnts, names }) {
```

Comment:
@hondahmd let's call it somehow else, this is a component, this should be named as that (eg. TabbedComponents)

Reply:
```suggestion
function TabbedComponents({ components }) {
```